src/pages/Shopping.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import Oggetti from '../Components/OggettiComponents/Oggetti'
+import AllCategories from '../Components/Filter/AllCategories'
+import Cart from '../Components/Cart/Cart'
+
+const Shopping = () => {
+    return (
+        <React.Fragment>
```

Comment:
The return value of a React component always has to be a single parent DOM node; in this case, we had two elements (`<div className='aside'>` and `<section>`) so in order to render them, we need to wrap them in a single element. I could've used a div, but I didn't want to affect the CSS of how those elements end up shown in the page by nesting them more. In such situations where you want to render separate elements in a single React component, you can wrap them in a special React element called `<React.Fragment>` to just tell React to group them together and pretend they are in one parent node.

Reply:
https://react.dev/reference/react/Fragment